### PR TITLE
[feat] integrate footer

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Integrate doppler ui system footer

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -2,20 +2,18 @@ import React from 'react';
...
-      </p>
+  <footer className="footer-main">
+    <div className="footer-wrapper">
+      <span>
+        <FormattedMessage id="footer.iso" />
+      </span>
+      <span>
+        &copy; {new Date().getFullYear()} Doppler LLC.{' '}
+        <FormattedMessage id="footer.allRightReserved" />.{' '}
+        <a className="link--default" href="https://fromdoppler.com/privacidad">
```
Comment (cbernat) [inline src/components/Footer/Footer.js]: Privacy link has english counterpart `https://www.fromdoppler.com/en/privacy-policy` you should consider adding this to language file.

**Comment by andresmoschini** [comment]
Nice! @fcoronelmakingsense did you compare it side by side with doppler footer?

If are there pending details, please create a ticket to improve it.

**Comment by andresmoschini** [comment]
image

If you use [Conventional commits](https://www.conventionalcommits.org) messages will be easy to generate and understand WebApp versions.

**Comment by fcoronelmakingsense** [comment]
> image
> 
> If you use [Conventional commits](https://www.conventionalcommits.org) messages will be easy to generate and understand WebApp versions.

Fixed

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (andresmoschini) [reply src/components/Footer/Footer.js]: @fcoronelmakingsense do not forget this.

**Pull request merged**